.lgtm.yml

```diff
@@ -0,0 +1,8 @@
+path_classifiers:
+  externs:
+    - "**/*.extern.js"
+extraction:
+  javascript:
+    index:
+      filters:
+        exclude: "examples"
```

Comment:
This won't quite work: there is a subtle difference between `filters/exclude` and plain `exclude`. The former expects a glob pattern, the latter a simple path. Since you want to exclude a path (and not a pattern) in this case, you can simply drop the `filters:` line.

Reply:
Done. Thanks!

Reply:
:+1: